Mercury-Store/Mercury-Store/Home/Views/BrandDetails/Cells/BrandProductsCollectionViewCell.swift

```diff
@@ -22,11 +23,8 @@ class BrandProductsCollectionViewCell: UICollectionViewCell {
     @IBOutlet weak private var currencyLabel: UILabel!
     
     @IBOutlet weak private var productForBrandPrice: UILabel!
-    public var isFavouriteProduct:Bool? {
-        willSet(newValue){
-            favouriteButton.favouriteState(state: newValue ?? false)
-        }
-    }
+    let viewModel:ProductCellViewModelType = ProductCellViewModel()
+    let disposeBag = DisposeBag()
     var item: Product? {
         didSet {
             guard let item = item else {
```

Comment:
Create method to update the view given a product.. Make didSet with minimal code lines.